src/contracts/core/StrategyManager.sol

```diff
@@ -150,7 +148,7 @@ contract StrategyManager is
         external
         initializer
     {
-        DOMAIN_SEPARATOR = keccak256(abi.encode(DOMAIN_TYPEHASH, bytes("EigenLayer"), block.chainid, address(this)));
+        DOMAIN_SEPARATOR = keccak256(abi.encode(DOMAIN_TYPEHASH, bytes("EigenLayer"), ORIGINAL_CHAIN_ID, address(this)));
```

Comment:
see comments in DelegationManager for matching logic in this contract

Reply:
Addressed: [3d1b89c](https://github.com/Layr-Labs/eigenlayer-contracts/pull/14/commits/3d1b89c1c2e0627bf2e5679d6e7588119f23a829)